From f634448b293ed232009b25c824f6dcd66961426f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B3hann=20B=2E=20Gu=C3=B0mundsson?= <johannbg@gmail.com>
Date: Fri, 14 May 2021 13:46:23 +0000
Subject: [PATCH] fix(packit): downstream has renamed the master branch to main

---
 .packit.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.packit.yml b/.packit.yml
index 8a995efe..482b4c4d 100644
--- a/.packit.yml
+++ b/.packit.yml
@@ -33,7 +33,7 @@ jobs:
 - job: propose_downstream
   trigger: release
   metadata:
-    dist_git_branches: master
+    dist_git_branches: main
 
 - job: tests
   trigger: pull_request

