From b9b6f0ee5b859a562e46a8c4e0dee0261fabf74d Mon Sep 17 00:00:00 2001
From: Lars Wendler <polynomial-c@gentoo.org>
Date: Fri, 14 May 2021 16:00:27 +0200
Subject: [PATCH] fix(dracut-util): print error message with trailing newline

Signed-off-by: Lars Wendler <polynomial-c@gentoo.org>
---
 src/util/util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/util/util.c b/src/util/util.c
index 8ae06d6c..b3498df6 100644
--- a/src/util/util.c
+++ b/src/util/util.c
@@ -123,7 +123,7 @@ static void usage(enum EXEC_MODE enumExecMode, int ret, char *msg)
 {
         switch (enumExecMode) {
         case UNDEFINED:
-                fprintf(stderr, "ERROR: 'dracut-util' has to be called via a symlink to the tool name.");
+                fprintf(stderr, "ERROR: 'dracut-util' has to be called via a symlink to the tool name.\n");
                 break;
         case GETARG:
                 fprintf(stderr, "ERROR: %s\nUsage: dracut-getarg <KEY>[=[<VALUE>]]\n", msg);

