From 5a720ebf6711a119649645de673bcb247e09a1ba Mon Sep 17 00:00:00 2001
From: Martin Wilck <mwilck@suse.de>
Date: Fri, 6 Mar 2020 10:26:43 +0100
Subject: [PATCH] dracut.spec: add 90nvdimm

---
 dracut.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dracut.spec b/dracut.spec
index 03b41471..0369dde6 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -364,6 +364,7 @@ install -m 0755 51-dracut-rescue-postinst.sh $RPM_BUILD_ROOT%{_sysconfdir}/kerne
 %{dracutlibdir}/modules.d/90lvm
 %{dracutlibdir}/modules.d/90mdraid
 %{dracutlibdir}/modules.d/90multipath
+%{dracutlibdir}/modules.d/90nvdimm
 %{dracutlibdir}/modules.d/90stratis
 %{dracutlibdir}/modules.d/90ppcmac
 %{dracutlibdir}/modules.d/90qemu

