From 791b2680d555ffa822e995423a90a38b7050b051 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 30 Aug 2011 16:22:46 +0200
Subject: [PATCH] 99base/init: save and restore environment given from the
 kernel

---
 modules.d/99base/init |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/modules.d/99base/init b/modules.d/99base/init
index 90128c7..4a91e53 100755
--- a/modules.d/99base/init
+++ b/modules.d/99base/init
@@ -8,6 +8,8 @@
 # Harald Hoyer <harald@redhat.com>
 # Jeremy Katz <katzj@redhat.com>
 
+export -p > /tmp/export.orig
+
 wait_for_loginit()
 {
     set +x
@@ -395,6 +397,8 @@ for i in $(export -p); do
             unset "$i";;
     esac
 done
+. /tmp/export.orig
+rm -f /tmp/export.orig
 
 initargs=""
 read CLINE </proc/cmdline
