From 21d60c38b40f9b35dde8f33e110bd540c32cfff1 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 9 Jul 2010 11:19:32 +0200
Subject: [PATCH] plymouth: depend on crypt, if cryptsetup exists

---
 modules.d/50plymouth/check |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules.d/50plymouth/check b/modules.d/50plymouth/check
index 838e7e2..ceb26a7 100755
--- a/modules.d/50plymouth/check
+++ b/modules.d/50plymouth/check
@@ -1,3 +1,3 @@
 #!/bin/bash
-[[ $1 = -d ]] && echo crypt
+[[ $1 = -d ]] && which cryptsetup &>/dev/null && echo crypt
 [[ -x /sbin/plymouthd && -x /bin/plymouth && -x /usr/sbin/plymouth-set-default-theme ]]
