From 224b47d83800fa04f1608f61caeb326a079beba3 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 22 Sep 2011 16:14:38 +0200
Subject: [PATCH] dracut-functions: inst_rules() add missing ""

---
 dracut-functions |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut-functions b/dracut-functions
index 18a2e89..5508809 100755
--- a/dracut-functions
+++ b/dracut-functions
@@ -474,7 +474,7 @@ inst_rules() {
     inst_dir "/lib/udev/rules.d"
     inst_dir "$_target"
     for _rule in "$@"; do
-        if [ "${rule#/}" = $rule ]; then
+        if [ "${rule#/}" = "$rule" ]; then
             for r in /lib/udev/rules.d /etc/udev/rules.d; do
                 if [[ -f $r/$_rule ]]; then
                     _found="$r/$_rule"
