From 2f3aa7537c646456eec33d59abd24e30d1cb7dce Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 18 Jul 2011 16:31:55 +0200
Subject: [PATCH] dracut.spec: fixed logrotate

---
 dracut.spec |    8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/dracut.spec b/dracut.spec
index 9af4418..f556c27 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -202,8 +202,8 @@ rm $RPM_BUILD_ROOT/sbin/mkinitrd
 rm $RPM_BUILD_ROOT/sbin/lsinitrd
 %endif
 
-mkdir -p $RPM_BUILD_ROOT/etc/logrotate.d/dracut
-install -m 0644 dracut.logrotate $RPM_BUILD_ROOT/etc/logrotate.d/dracut
+mkdir -p $RPM_BUILD_ROOT/etc/logrotate.d
+install -m 0644 dracut.logrotate $RPM_BUILD_ROOT/etc/logrotate.d/dracut_log
 
 %clean
 rm -rf $RPM_BUILD_ROOT
@@ -228,7 +228,6 @@ rm -rf $RPM_BUILD_ROOT
 %config(noreplace) /etc/dracut.conf.d/01-dist.conf
 %endif
 %dir /etc/dracut.conf.d
-%config(noreplace) /etc/logrotate.d/dracut
 %{_mandir}/man8/dracut.8*
 %{_mandir}/man7/dracut.kernel.7*
 %{_mandir}/man5/dracut.conf.5*
@@ -261,8 +260,7 @@ rm -rf $RPM_BUILD_ROOT
 %{_datadir}/dracut/modules.d/98selinux
 %{_datadir}/dracut/modules.d/98syslog
 %{_datadir}/dracut/modules.d/99base
-# logfile needs no logrotate, because it gets overwritten
-# for every dracut run
+%config(noreplace) /etc/logrotate.d/dracut_log
 %attr(0644,root,root) %ghost %config(missingok,noreplace) %{_localstatedir}/log/dracut.log
 %dir %{_sharedstatedir}/initramfs
 
-- 
1.7.9.3

