From f863c2c020e2509b6562b11484aac1ef41409101 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 17 Nov 2011 11:01:41 +0100
Subject: [PATCH] git2spec.pl: strip path from patches

---
 git2spec.pl |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/git2spec.pl b/git2spec.pl
index 27e5889..38f899a 100755
--- a/git2spec.pl
+++ b/git2spec.pl
@@ -49,6 +49,7 @@ while(<>) {
 	print $_;
 	$num=1;
 	for(@patches) {
+	    s/.*\///g;
 	    print "Patch$num: $_";
 	    $num++;
 	}
