From a3e125cb6c306eca510fcc508a2e812d172de977 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 17 Jun 2010 10:45:39 +0200
Subject: [PATCH 073/133] dracut.spec: moved znet to dracut-network

---
 dracut.spec |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/dracut.spec b/dracut.spec
index a25b92f..b47d6d2 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -186,7 +186,6 @@ rm -rf $RPM_BUILD_ROOT
 %{_datadir}/dracut/modules.d/95dasd
 %{_datadir}/dracut/modules.d/95dasd_mod
 %{_datadir}/dracut/modules.d/95zfcp
-%{_datadir}/dracut/modules.d/95znet
 %{_datadir}/dracut/modules.d/95terminfo
 %{_datadir}/dracut/modules.d/95udev-rules
 %{_datadir}/dracut/modules.d/95uswsusp
@@ -205,6 +204,7 @@ rm -rf $RPM_BUILD_ROOT
 %{_datadir}/dracut/modules.d/95nbd
 %{_datadir}/dracut/modules.d/95nfs
 %{_datadir}/dracut/modules.d/45ifcfg
+%{_datadir}/dracut/modules.d/95znet
 
 %files fips
 %defattr(-,root,root,0755)
-- 
1.7.3

