From 21644eb9df66462ae8644510921aad44fe654595 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 4 Jun 2012 14:00:49 +0200
Subject: [PATCH] dracut.sh: precopy some essential device nodes

---
 dracut.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dracut.sh b/dracut.sh
index eb58f78..16ff0e5 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -700,6 +700,9 @@ if [[ $kernel_only != yes ]]; then
     for _d in $hookdirs; do
         mkdir -m 0755 -p ${initdir}/lib/dracut/hooks/$_d
     done
+    if [[ "$UID" = "0" ]]; then
+        cp -a /dev/kmsg /dev/null /dev/console $initdir/dev
+    fi
 fi
 
 mkdir -p "$initdir/.kernelmodseen"
