From e90220701445dc58c56c3df22276cbc9b1299dc2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ga=C3=ABl=20PORTAY?= <gael.portay@collabora.com>
Date: Tue, 24 Sep 2019 22:39:43 -0400
Subject: [PATCH] dracut.modules.7.asc: fix another typo

---
 dracut.modules.7.asc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.modules.7.asc b/dracut.modules.7.asc
index 67a56bd4..fb30409e 100644
--- a/dracut.modules.7.asc
+++ b/dracut.modules.7.asc
@@ -159,7 +159,7 @@ check():
 return 0
 ----
 
-The we create the install() function, which installs a cmdline hook with
+Then we create the install() function, which installs a cmdline hook with
 priority number 20 called _parse-insmodpost.sh_. It also installs the
 _insmodpost.sh_ script in _/sbin_.
 

