From c41b09663f4bd28594ec3a0b341b6ebbb20ee34e Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 23 Aug 2010 11:01:04 +0200
Subject: [PATCH 117/133] dm: install all md/dm* kernel modules

---
 modules.d/90dm/installkernel |    8 ++------
 1 files changed, 2 insertions(+), 6 deletions(-)

diff --git a/modules.d/90dm/installkernel b/modules.d/90dm/installkernel
index af51a22..f5a8ad9 100755
--- a/modules.d/90dm/installkernel
+++ b/modules.d/90dm/installkernel
@@ -1,9 +1,5 @@
 #!/bin/bash
 
-for i in \
-    dm-snapshot dm-delay dm-mirror dm-mod dm-queue-length \
-    dm-region-hash dm-round-robin dm-service-time dm-snapshot \
-    ; do
-    modinfo -k $kernel $i >/dev/null 2>&1 && instmods $i
-done
+instmods =drivers/md
+
 
-- 
1.7.3

