From 55102c72b00f1aba6d6e7ea8b90f45f3f0a6726d Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 23 Aug 2010 14:42:33 +0200
Subject: [PATCH 119/133] dmraid: switch to rd_NO_MDIMSM, if no mdadm installed

---
 modules.d/90dmraid/parse-dm.sh |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/modules.d/90dmraid/parse-dm.sh b/modules.d/90dmraid/parse-dm.sh
index f031569..63780bd 100644
--- a/modules.d/90dmraid/parse-dm.sh
+++ b/modules.d/90dmraid/parse-dm.sh
@@ -3,3 +3,9 @@ if getarg rd_NO_DM || getarg nodmraid; then
     info "rd_NO_DM: removing DM RAID activation"
     udevproperty rd_NO_DM=1
 fi
+
+if [ ! -x /sbin/mdadm ] || getarg rd_NO_MDIMSM || getarg noiswmd; then
+    info "rd_NO_MDIMSM: no MD RAID for imsm/isw raids"
+    udevproperty rd_NO_MDIMSM=1
+fi
+
-- 
1.7.3

