From 5da70385bcece0655bed4c408214173de51768ee Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B3hann=20B=2E=20Gu=C3=B0mundsson?= <johannbg@gmail.com>
Date: Fri, 7 Aug 2020 19:16:46 +0000
Subject: [PATCH] Removing dracut entry not compat with new plugin

---
 .github/labeler.yml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/.github/labeler.yml b/.github/labeler.yml
index 61eb79f6..6e232f3f 100644
--- a/.github/labeler.yml
+++ b/.github/labeler.yml
@@ -294,6 +294,3 @@ uefi-lib:
 test:
   - test/*
   - test/**/*
-
-dracut:
-  - any: ['!.github/*', '!.mkosi/*', '!install/*', '!examples/*', '!modules.d/*', '.test/*']

