From 1855efb15af9009c1f6db77c1d9ce012129bcb71 Mon Sep 17 00:00:00 2001
From: Daniel Molkentin <dmolkentin@suse.com>
Date: Tue, 26 Nov 2019 12:29:47 +0100
Subject: [PATCH] Document initqueue/online hook

---
 dracut.modules.7.asc | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/dracut.modules.7.asc b/dracut.modules.7.asc
index 81135cff..3c056430 100644
--- a/dracut.modules.7.asc
+++ b/dracut.modules.7.asc
@@ -93,13 +93,18 @@ udev state.
 
 ==== Initqueue settled
 
-This hooks (initqueue/settled) gets executed every time udev has settled.
+This hook (initqueue/settled) gets executed every time udev has settled.
 
 ==== Initqueue timeout
 
-This hooks (initqueue/timeout) gets executed, when the main loop counter becomes
+This hook (initqueue/timeout) gets executed, when the main loop counter becomes
 half of the rd.retry counter.
 
+==== Initqueue online
+
+This hook (initqueue/online) gets executed whenever a network interface comes online
+(that is, once it is up and configured by the configured network module).
+
 ==== Initqueue finished
 
 This hook (initqueue/finished) is called after udev has settled and

