From 9eb1d1ed5ed1eb5008e6173c1ae0480132a4bea4 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 2 Oct 2020 13:40:00 +0200
Subject: [PATCH] dracut.spec: remove fedora pre 30 quirks

---
 dracut.spec | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/dracut.spec b/dracut.spec
index 3597c6e5..fded5f17 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -279,7 +279,7 @@ rm -f -- $RPM_BUILD_ROOT%{_mandir}/man1/lsinitrd.1*
 echo 'hostonly="no"' > $RPM_BUILD_ROOT%{dracutlibdir}/dracut.conf.d/02-generic-image.conf
 echo 'dracut_rescue_image="yes"' > $RPM_BUILD_ROOT%{dracutlibdir}/dracut.conf.d/02-rescue.conf
 
-%if 0%{?fedora} <= 30 || 0%{?rhel} <= 8
+%if 0%{?rhel} > 0 && 0%{?rhel} <= 8
 mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/kernel/postinst.d
 install -m 0755 51-dracut-rescue-postinst.sh $RPM_BUILD_ROOT%{_sysconfdir}/kernel/postinst.d/51-dracut-rescue-postinst.sh
 %endif
@@ -478,8 +478,7 @@ install -m 0755 51-dracut-rescue-postinst.sh $RPM_BUILD_ROOT%{_sysconfdir}/kerne
 %files config-rescue
 %{dracutlibdir}/dracut.conf.d/02-rescue.conf
 %{_prefix}/lib/kernel/install.d/51-dracut-rescue.install
-%if 0%{?fedora} <= 30 || 0%{?rhel} <= 8
-# FIXME: remove after F30
+%if 0%{?rhel} > 0 && 0%{?rhel} <= 8
 %{_sysconfdir}/kernel/postinst.d/51-dracut-rescue-postinst.sh
 %endif
 

